### ISSUE TITLE
Unify hardware drawing

### ISSUE THREAD

**Issue by willglynn** [raceintospace/raceintospace#102]
The game currently uses three 320x200 images (stored in `CIA.BUT` and `VAB.IMG`) for different hardware sprites:

![cia.but](https://f.cloud.github.com/assets/118362/63525/33862c7a-5dea-11e2-9be1-262f16bf2fe8.png)
![vab.img.0](https://f.cloud.github.com/assets/118362/63526/339368a4-5dea-11e2-86ed-e9921c983a96.png)
![vab.img.1](https://f.cloud.github.com/assets/118362/63527/339b3c6e-5dea-11e2-88c8-a5a0e567844b.png)

There's hardcoded offsets all over, the images are difficult or impossible to change, and there's a lot of duplication. We should replace these with a class that represents a drawable type of hardware.

**Comment by peyre** [comment]
I've looked these over a bit, even made some improvements to the rockets (esp. the N1).  A couple things I notice about these is that the VAB images have a shadow that the CIA ones don't, and the VAB ones also show cutouts of the capsule/shuttle interiors that the CIA ones lack.  I'm inclined to think these should stay separate.

**Comment by peyre** [comment]
Of course, that doesn't mean we couldn't pull the images out into discrete files, rather than having the game display coordinates a,b-x,y from one of these files.
